tests/OmniSharp.Roslyn.CSharp.Tests/BlockStructureFacts.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using System.Threading.Tasks;
+using OmniSharp.Models.v2;
+using OmniSharp.Roslyn.CSharp.Services.Structure;
+using OmniSharp.Roslyn.Extensions;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Roslyn.CSharp.Tests
+{
+    public class BlockStructureFacts : AbstractSingleRequestHandlerTestFixture<BlockStructureService>
+    {
+        public BlockStructureFacts(ITestOutputHelper output, SharedOmniSharpHostFixture sharedOmniSharpHostFixture)
+            : base(output, sharedOmniSharpHostFixture)
+        {
+        }
+
+        protected override string EndpointName => OmniSharpEndpoints.V2.BlockStructure;
+
+        [Fact]
+        public async Task UsesRoslynBlockStructureService()
+        {
+            var testFile = new TestFile("foo.cs", @"class Foo[|
+{
+    void M()[|
+    {
+        if (false)[|
```

Comment:
Note that this isn't collapsible in Visual Studio. I think we might be conflating the vertical indicators in VS with code folding.

Reply:
Well, actually 😄.   : 
![image](https://user-images.githubusercontent.com/3751401/40937370-4617f724-67f3-11e8-8b24-d90c7fdab85b.png)